MinPlatformPkg/Test/TestPointStubDxe/TestPointStubDxeMm.inf

```diff
@@ -9,13 +10,13 @@
 [Defines]
   INF_VERSION                    = 0x00010005
   BASE_NAME                      = TestPointStubDxe
-  FILE_GUID                      = C034A2CB-1190-4638-9DDF-6F297824741E
+  FILE_GUID                      = 6D515FCA-4311-4034-A5B9-F4065B412461
```

Comment:
Somehow this is a rename of the file? Does this GUID really need to be changed?

Reply:
This must have been left over for when I had a separate file for Standalone MM and Smm.  I'll change this back.

Reply:
Updated.